src/helpers/git-sourcemaps-repository-data.ts

```diff
@@ -2,7 +2,7 @@ import fs from 'fs'
 import * as simpleGit from 'simple-git'
 import {Writable} from 'stream'
 import {URL} from 'url'
-import {renderGitWarning, renderSourcesNotFoundWarning} from './renderer'
+import {renderGitWarning, renderSourcesNotFoundWarning} from '../commands/sourcemaps/renderer'
```

Comment:
As much as possible, common helpers should include functions from commands, nor should they write to standard output as it is specific to each command.

Could you remove these rendering functions?
I expect `getRepositoryData` could just not catch the exception and have it handled by the calling function (within the command, which has access to rendering functions) and `matchSourcemap` could return an empty array and the array content should be handled in `upload.ts`.